Blasphemous.Framework.Levels/LevelFramework.cs

```diff
@@ -297,14 +298,23 @@ protected override void OnRegisterServices(ModServiceProvider provider)
             new SceneLoader("D17Z01S10_LOGIC", "LOGIC/INTERACTABLES/{0}"),
             new BloodModifier()));
 
-        // Traps
-        provider.RegisterObjectCreator("spikes", new ObjectCreator(
+        // spikes
+        provider.RegisterObjectCreator("spikes-wasteland", new ObjectCreator(
             new SceneLoader("D01Z03S01_DECO", "MIDDLEGROUND/AfterPlayer/Spikes/{0}"),
             new SpikeModifier()));
 
         // Misc.
         provider.RegisterObjectCreator("lantern", new ObjectCreator(
             new SceneLoader("D20Z01S02_LOGIC", "LOGIC/INTERACTABLES/Chain Hook"),
             new NoModifier("Lantern")));
+        provider.RegisterObjectCreator("bell-face", new ObjectCreator(
+            new SceneLoader("D03Z02S06_LOGIC", "TRAPS/TRAP_SHOCK_ENEMY"),
+            new NoModifier("Face bell")));
+        provider.RegisterObjectCreator("bell-iron", new ObjectCreator(
+            new SceneLoader("D03Z02S06_LOGIC", "TRAPS/TRAP_SHOCK_CHAIN_REACTION"),
+            new NoModifier("Iron bell")));
+        provider.RegisterObjectCreator("ladder-jondo", new ObjectCreator(
```

Comment:
You added a ladder object but there is no class called "LadderModifier".  Go ahead and remove this, as I dont think ladders belong in the framework yet